README.md

```diff
@@ -19,7 +19,7 @@ None.
 
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
-| apt_source_files | A list of apt source files to modify.  Files that do not exist will be ignored. | `[/etc/apt/sources.list, /etc/apt/sources.list.d/debian.sources]` | No |
+| apt_over_https_apt_source_files | A list of apt source files to modify.  Files that do not exist will be ignored. | `[/etc/apt/sources.list, /etc/apt/sources.list.d/debian.sources]` | No |
```

Comment:
Just a thought, but since apt is already in the role name now that we're prepending it to variables, would `apt_over_https_source_files` be a little less redundant than `apt_over_https_apt_source_files`?

Reply:
I'd probably just leave it as `apt_over_https_apt_source_files` since it is more descriptive, but I'm fine either way.

Reply:
I agree with @dav3r.  `apt_over_https_source_files` sounds like the source code for something called "APT over HTTPS" and only saves four characters.  I think the extra descriptiveness is worth four characters.

Reply:
I have been outvoted, and will now leave the island